demos/js/console.js

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright 2022, Tony Atkins
+ * Copyright 2011-2020, Colin Clark
+ *
+ *  Licensed under the MIT license, see LICENSE for details.
+ */
+(function (fluid) {
+    "use strict";
+    var youme = fluid.registerNamespace("youme");
+
+    fluid.defaults("youme.demos.console.textareaWithError", {
+        gradeNames: ["youme.templateRenderer"],
+        markup: {
+            container: "<div class='console-textarea-with-error'><textarea class='editor'>%editorString</textarea><div class='error'>%errorString</div></div>"
+        },
+
+        model: {
+            errorString: false,
+            editorString: ""
+        },
+
+        selectors: {
+            error: ".error",
+            editor: ".editor"
+        },
+
+        modelListeners: {
+            errorString: [
+                {
+                    excludeSource: "init",
```

Comment:
This could be a new-style materialised relay:
````
modelRelay: {
    errorText: {
        target: "dom.error.text",
        source: "{that}.model.errorString"
    }
}
````

Reply:
Done.